unraid_mcp/tools/unraid.py

```
@@ -0,0 +1,1891 @@
+                "success": True,
+                "subaction": subaction,
+                "idempotent": True,
+                "message": f"Container already in desired state for '{subaction}'",
+            }
+        return {
+            "success": True,
+            "subaction": subaction,
+            "container": (data.get("docker") or {}).get(subaction),
+        }
+
+    raise ToolError(f"Unhandled docker subaction '{subaction}' — this is a bug")
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)